### DISCUSSION TITLE
Firewall port

### DISCUSSION THREAD

**Discussion by rgtripathi** [microsoft/aisdkforsapabap#24] [General]
After installing AI SDK for ABAP on the SAP system, what should be the port that should be opened up on the firewall ringfencing the server running SAP to allow outbound traffic to Azure OpenAI endpoint? Will this traffic be on port 443 being HTTPS? @gopalnair

**Comment by gopalnair** [comment]
Hi @rgtripathi ,
   Typically, 443 is the HTTPS port and will usually be open - unless you are not using HTTPS at all in any other services (its an unusual scenario). So, if you have not already configured firewalls to allow HTTPS traffic, I would start there for potential communication issue resolution.

Having said that, the other part is certificates. If you refer to documentation/videos, you will notice the steps to get and install relevant certificates in SAP using STRUST.

Best Regards,
Gopal Nair.

**Reply by rgtripathi** [reply]
Thanks Gopal
